### DISCUSSION TITLE
spf.lua fails

### DISCUSSION THREAD

**Discussion by M-Stenzel** [rspamd/rspamd#4898] [General]
Hi group,
running version 3.8.4 on a Linux system (no docker), lua version 5.4.6 I receive the following in the logs

`init of /usr/share/rspamd/lualib/lua_ffi/spf.lua failed: /usr/share/rspamd/lualib/lua_ffi/spf.lua:71: size of C type is unknown or too large at line 35; trace: [1]:{[C]:-1 - cdef [C]}; [2]:{/usr/share/rspamd/lualib/lua_ffi/spf.lua:71 - <unknown> [main]};`

Will you please verify? I do not have an unusual configuration.

Martin.

**Comment by fatalbanana** [comment]
I suppose you are using SuSE. https://github.com/rspamd/rspamd/discussions/4819

[The apparent reason is that PLUGINSDIR is incorrectly set at build time and so Rspamd tries to execute things it shouldn't as plugins](https://github.com/rspamd/rspamd/issues/4977#issuecomment-2227294583)

[The solution could be to avoid setting it entirely](https://github.com/rspamd/rspamd/discussions/4819#discussioncomment-10417508)

**Reply by M-Stenzel** [reply]
Bingo, I contacted the package maintainer (very recently).
But there is nothing special about SuSE. I do not think that there is an interfering patch.
There are some more bug reports with that error message (not related to SuSE, not related to rspamd).

**Reply by mcepl** [reply]
(Overworked) Lua SUSE maintainer here: any reasonable submit requests changing the packaging in openSUSE are more than welcome.
